### DISCUSSION TITLE
ProfileMenu close after click

### DISCUSSION THREAD

**Discussion by franklupo** [microsoft/fluentui-blazor#2421] [General]
How to close ProfileMenu es. click on `View account`?

![image](https://github.com/user-attachments/assets/326b160e-45eb-4933-818d-e175d681e734)

best regards

**Comment by vnbaaij** [comment]
Click outside of the ProfileMenu. Or click on the 'open icon' again (not sure about that one without looking at the source)

**Reply by franklupo** [reply]
Ok, 
But for example you click `View account` and open new page, the ProfileMenu should close.

**Reply by vnbaaij** [reply]
Agree.

**Reply by franklupo** [reply]
it does not work, remains open. It would take a properity to close and then it will be managed with lost focus event

**Reply by dvoituron** [reply]
I'm not sure if the ProfileMenu needs to be closed in this case.

For example, when you click on "Manage Account", a new page opens and the menu is still displayed.
https://www.bing.com/chat 
![image](https://github.com/user-attachments/assets/4d6b0bfc-9238-4310-ac3c-2e02ccc530ac)
